gamestonk_terminal/cryptocurrency/defi/smartstake_view.py

```diff
@@ -0,0 +1,109 @@
+"""SentimentInvestor View"""
+__docformat__ = "numpy"
+
+import os
+import matplotlib.dates as mdates
+from matplotlib import pyplot as plt
+
+from gamestonk_terminal.cryptocurrency.defi import smartstake_model
+from gamestonk_terminal.helper_funcs import export_data, print_rich_table
+from gamestonk_terminal import feature_flags as gtff
+
+# pylint: disable=E1101
+
+LUNA_CIR_SUPPLY_CHANGE = "lunaSupplyChallengeStats"
```

Comment:
Are there any other supply types ? If yes does it make sense to also add them ? And then user could specify in command which kind of supply type he is intertested in ?

Reply:
You're spot on at this :) Yes, there are others and the plan is to incorporate them all. However, as mentioned in the Blocking point, I suspect they update the public key and token on a daily basis. So I will send them a message first to see if we can have a fixed ones. Otherwise, if they keep on update it every day, we won't be able to update the key & tokens. So I'm waiting for that, and if that's okay, I'll incorporate rest of the supply_type